### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by tienthegainz**
Ticket: https://edu-redmine.sun-asterisk.vn/issues/28024

### CODE REVIEW

**Comment by vanvtt-0952** [comment]
ready

**Review by drnguyenn — APPROVED** [review]

**Review by qngdt — APPROVED** [review]

**Comment by tienthegainz** [comment]
ready

**Comment by tienthegainz** [comment]
ready

**Comment by tienthegainz** [comment]
ready

**Comment by tienthegainz** [comment]
ready

**Comment by tienthegainz** [comment]
ready

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		<finalName>project1</finalName>
+		<plugins>
+			<!-- <plugin>
+				<artifactId>maven-war-plugin</artifactId>
+				<version>${maven-war-plugin.version}</version>
+				<configuration>
+					<warSourceDirectory>WebContent</warSourceDirectory>
+					<failOnMissingWebXml>false</failOnMissingWebXml>
+				</configuration>
+			</plugin> -->
```
Comment (vanvtt-0952) [inline pom.xml]: manhj tay xóa đi gái

File: src/main/webapp/WEB-INF/configs/spring-service.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+	<!-- Declare Service here-->
+	<!--		TODO -->
+<!--	<bean id="StudentService" parent="baseTransactionProxy">-->
+<!--		<property name="target">-->
+<!--			<bean class="app.service.impl.StudentServiceImpl">-->
+<!--				<property name="studentDAO" ref="StudentDAO" />-->
+<!--			</bean>-->
+<!--		</property>-->
+<!--	</bean>-->
```
Comment (vanvtt-0952) [inline src/main/webapp/WEB-INF/configs/spring-service.xml]: Xóa code comment đi nè

Comment (tienthegainz) [reply src/main/webapp/WEB-INF/configs/spring-service.xml]: Ok rồi ạ

**Comment by tienthegainz** [comment]
ready

**Pull request merged**